docs/index.md

```diff
@@ -62,3 +63,4 @@ PowerShell Module                                    | Version   | Publisher
 [psgallery-module-poshssh]: https://www.powershellgallery.com/packages/Posh-SSH
 [developer-module-powercli]: https://developer.vmware.com/tool/vmware-powercli
 [github-module-poshssh]: https://github.com/darkoperator/Posh-SSH
+[Shut down a VxRail cluster]: https://www.dell.com/support/manuals/en-us/vxrail-appliance-series/vxrail-8.x_admin_guide/shut-down-a-vxrail-cluster?guid=guid-da69fd52-38b2-465e-b8d9-45191b016679&lang=en-us
```

Comment:
```suggestion
[vxrail-cluster-shutdown]: https://www.dell.com/support/manuals/en-us/vxrail-appliance-series/vxrail-8.x_admin_guide/shut-down-a-vxrail-cluster?guid=guid-da69fd52-38b2-465e-b8d9-45191b016679&lang=en-us
```

Reply:
addressed in commit f8c5a69ef84a1a40675a91879ad0a67701fba108